src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -205,21 +205,22 @@
          token-networks-ids                        (mapv #(:chain-id %) token-networks)
          token-not-supported-in-receiver-networks? (not-any? (set receiver-networks)
                                                              token-networks-ids)]
-     {:db (cond-> db
-            :always      (update-in [:wallet :ui :send] dissoc :collectible)
-            :always      (assoc-in [:wallet :ui :send :token-display-name]
-                          (:symbol token))
-            :always      (assoc-in
-                          [:wallet :ui :send :token-not-supported-in-receiver-networks?]
-                          token-not-supported-in-receiver-networks?)
-            token        (assoc-in [:wallet :ui :send :token] token)
-            token-symbol (assoc-in [:wallet :ui :send :token-symbol] token-symbol))
-      :fx [[:dispatch [:wallet/clean-suggested-routes]]
-           [:dispatch
-            [:wallet/wizard-navigate-forward
-             {:current-screen stack-id
-              :start-flow?    start-flow?
-              :flow-id        :wallet-send-flow}]]]})))
+     (when (or token token-symbol)
+       {:db (cond-> db
+              :always      (update-in [:wallet :ui :send] dissoc :collectible)
+              :always      (assoc-in
+                            [:wallet :ui :send :token-not-supported-in-receiver-networks?]
+                            token-not-supported-in-receiver-networks?)
+              token        (assoc-in [:wallet :ui :send :token] token)
+              token        (assoc-in [:wallet :ui :send :token-display-name]
```

Comment:
The only change here is that `:token-display-name` is now set only if `token` is non-nil. Before it was set always but was `nil`

Reply:
Please just make sure the send flow is not broken if we start it from any possible entry point

Reply:
There are 2 possible entry points to initiate this event - when the user selects a token from the wallet screen or when selects an asset after pressing the "send" button within a specific account. I checked both, they work without changes.